development/extensions/new_in_rhea.rst

```diff
@@ -98,15 +98,13 @@ Fortunately, the integration is pretty seamless and most existing extensions tha
   * ``generate_text_for_edit()``
   * ``generate_text_for_storage()``
   * ``strip_bbcode()``
-  * ``smiley_text()``
```

Comment:
I see what you're saying.

FYI if you use its force_option it did make smilies appear in their text form instead of the image form:
https://github.com/phpbb/phpbb/blob/master/phpBB/includes/functions_content.php#L1040-L1043

It has not been deprecated for B.C. reasons. But I see that it has no use for 3.2 parsed text content, so I guess it is OK to remove from here.